bin/genesis

```diff
@@ -1557,16 +1488,8 @@ cmd_new_site() {
 		echo "Created site ${name} (from template ${template}):"
 
 	else
-		case ${DEPLOYMENT_TYPE} in
-		(normal)
-			create_normal_site ${name}
-			echo "Created site ${name}:"
-			;;
-		(bosh-init)
-			create_bosh_init_site ${name}
-			echo "Created (bosh-init) site ${name}:"
-			;;
-		esac
+		create_normal_site ${name}
```

Comment:
Addressed this, and the other rename

> On Jun 28, 2016, at 10:58 AM, James Hunt <notifications@github.com> wrote:
> 
> In bin/genesis <https://github.com/starkandwayne/genesis/pull/41#discussion_r68773709>:
> 
> > @@ -1557,16 +1488,8 @@ cmd_new_site() {
> >  		echo "Created site ${name} (from template ${template}):"
> >  
> >  	else
> > -		case ${DEPLOYMENT_TYPE} in
> > -		(normal)
> > -			create_normal_site ${name}
> > -			echo "Created site ${name}:"
> > -			;;
> > -		(bosh-init)
> > -			create_bosh_init_site ${name}
> > -			echo "Created (bosh-init) site ${name}:"
> > -			;;
> > -		esac
> > +		create_normal_site ${name}
> Should this get renamed to just create_site?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/starkandwayne/genesis/pull/41/files/f2605cc21fbaccc97b1b5a37ee3e11b1c05cc7d8#r68773709>, or mute the thread <https://github.com/notifications/unsubscribe/AG2SUmhKmm-CBlU5rTtCrU2W39LsU3bJks5qQTaIgaJpZM4JAMHN>.
>